python/cuda_parallel/cuda/parallel/experimental/iterators/_iterators.py

```diff
@@ -463,3 +473,39 @@ def _get_last_element_ptr(device_array) -> int:
 
     ptr = get_data_pointer(device_array)
     return ptr + offset_to_last_element
+
+
+def _replace_duplicate_values(*ds, replacement_value):
+    # given a sequence of dictionaries, return a sequence of dictionaries
+    # such that for any found duplicate keys, the value is set to `scrub_value`.
```

Comment:
```suggestion
    # such that for any found duplicate keys, the value is set to `replacement_value`. 
```

Or should we rename replacement_value` to `scrub_value`?

Anyhow, the last function argument is not being used in the body of the function.